RadarSDK/RadarLocationManager.m

```diff
@@ -337,8 +346,11 @@ - (void)updateTracking:(CLLocation *)location {
         } else {
             [self stopUpdates];
             [self removeAllRegions];
-            [self.locationManager stopMonitoringVisits];
-            [self.locationManager stopMonitoringSignificantLocationChanges];
+            
+            if (!fromInitialize) {
```

Comment:
Let's add a comment here explaining why we're being careful about shutting down monitoring on the `CLLocationManager`.